docker-compose.yml

```diff
@@ -1,5 +1,14 @@
-scan:
-  env_file: .env
-  build: .
-  volumes:
-    - ./:/home/scanner
+version: '3.3'
+
+services:
+  scan:   
+    build: .
+    volumes:
+      - .:/home/scanner
+    # Some internet providers intercept NXDOMAIN and show a search
+    # page instead.  Google's DNS does not, so I prefer to use it.
+    dns:
+      - 8.8.8.8
```

Comment:
I wonder if we could provide DNS server config as a `pshtt` feature flag and handle it in app-layer code...